JustSaying.UnitTests/JustSaying.UnitTests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I would also like to consolidate the main projects - there's too many of them, especially now that they produce individual nuget packages.